src/app/lib/mainsail/transaction.service.ts

```diff
@@ -82,26 +82,26 @@ export class TransactionService extends Services.AbstractTransactionService {
 		}
 	}
 
-	public override async transfer(input: Services.TransferInput): Promise<Contracts.SignedTransactionData> {
+	public override async transfer(input: Services.TransferInput): Promise<SignedTransactionData> {
 		applyCryptoConfiguration(this.#configCrypto);
 		this.#assertGasFee(input);
 		this.#assertAmount(input);
 
-		const transaction = this.#app.resolve(EvmCallBuilder);
-
 		const { address } = await this.#signerData(input);
 		const nonce = await this.#generateNonce(address, input);
 
-		transaction
-			.network(this.#configCrypto.crypto.network.chainId)
-			.gasLimit(input.gasLimit)
-			.recipientAddress(input.data.to)
-			.payload("")
-			.nonce(nonce)
-			.value(parseUnits(input.data.amount, "ark").valueOf())
-			.gasPrice(parseUnits(input.gasPrice, "gwei").toNumber());
+		const { transaction } = (
+			await TransactionBuilder.new({ value: parseUnits(input.data.amount, "ark").valueOf() })
+				.payload("")
+				.recipientAddress(input.data.to)
```

Comment:
```suggestion
			await TransactionBuilder.new()
				.value(parseUnits(input.data.amount, "ark").valueOf())
				.recipientAddress(input.data.to)
```

does the same but easier to digest